samples/client/petstore/python/test/__init__.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
```

Comment:
Do we still need this file?
I think that we only made it so we could import the TestCategory.make_instance into the test_pet.py file an other similar parent child use cases.
If we don't need it can you remove it?

Reply:
Correct.Done.